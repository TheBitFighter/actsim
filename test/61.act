defproc test()
{
  bool x, y;
  spec {
    rand_init (x)
  }
  prs {
   x -> y-
  ~x -> y+
  } 
}


