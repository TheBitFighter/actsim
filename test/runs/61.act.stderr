In expanding test<> (61.act:1)
WARNING: Process `test<>': local variable `x': no driver
